presto-hive/src/main/java/com/facebook/presto/hive/util/Statistics.java

```diff
@@ -96,6 +103,25 @@ private static OptionalLong reduce(OptionalLong first, OptionalLong second, Redu
         return OptionalLong.empty();
     }
 
+    public static <T> List<T> pickRandomSample(List<T> elements, int sampleSize)
+    {
```

Comment:
Also this method has nothing to do with `Statistics`. It is very generic, so it could be moved to something like `PrestoCollections` or `PrestoLists`.